workers/unity/Assets/Playground/Config/PlayerTemplate.cs

```diff
@@ -25,7 +33,8 @@ public static EntityTemplate CreatePlayerEntityTemplate(string workerId, Improba
             PlayerLifecycleHelper.AddPlayerLifecycleComponents(template, workerId, clientAttribute,
                 WorkerUtils.UnityGameLogic);
 
-            template.SetReadAccess(WorkerUtils.UnityClient, WorkerUtils.UnityGameLogic, WorkerUtils.AndroidClient, WorkerUtils.iOSClient);
+            template.SetReadAccess(WorkerUtils.UnityClient, WorkerUtils.UnityGameLogic, WorkerUtils.AndroidClient,
```

Comment:
I didn't know we had this line :S. This means this only works if you have the same setup as playground.